src/service/rooms/event_handler/mod.rs

```diff
@@ -1645,6 +1645,20 @@ impl Service {
 
     /// Returns Ok if the acl allows the server
     pub fn acl_check(&self, server_name: &ServerName, room_id: &RoomId) -> Result<()> {
+        if !services()
```

Comment:
Can we verify if this will override room ACLs? (looks to be the code below for the room ACL event)

If so then this needs to be changed to still respect room ACLs so this won't be used as a way to ignore room ACLs.

Reply:
this respects room ACLs, as in if the server blocks something it will block it but otherwise it'll check if the room allows something before it lets it through